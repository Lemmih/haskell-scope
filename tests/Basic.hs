module Basic where

binder = binder
