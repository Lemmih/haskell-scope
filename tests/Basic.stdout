Scope errors:

Definitions:
  Definition 1, from Basic.decl0:
    [4mbinder[0m = binder

Use sites:
  Definition used: 1
    binder = [4mbinder[0m
